android/build.gradle

```diff
@@ -18,10 +18,13 @@ rootProject.allprojects {
     repositories {
         google()
         mavenCentral()
+        // just a test
+        maven { url 'https://storage.googleapis.com/download.flutter.io' }
```

Comment:
what is this for?

Reply:
This was a test to provide a dependency so that I don't need to edit the code in two Android Studio instances, it makes trouble as I see so I'll revert this